emojis.json

```diff
@@ -3656,13 +3656,13 @@
   "skier": {
     "keywords": ["sports", "winter", "snow"],
     "char": "⛷",
-    "fitzpatrick_scale": true,
+    "fitzpatrick_scale": false,
     "category": "activity"
   },
   "snowboarder": {
     "keywords": ["sports", "winter"],
     "char": "🏂",
-    "fitzpatrick_scale": true,
+    "fitzpatrick_scale": false,
```

Comment:
https://emojipedia.org/snowboarder-type-1-2/ This should be true shouldn't it?

Reply:
Yes